src/main/java/com/rocketden/main/exception/RoomError.java

```diff
@@ -14,7 +14,7 @@
     NO_HOST(HttpStatus.BAD_REQUEST, "There is no host provided."),
     NOT_FOUND(HttpStatus.NOT_FOUND, "A room could not be found with the given id."),
     DUPLICATE_USERNAME(HttpStatus.CONFLICT, "A user with the nickname provided has already joined the room."),
-    ALREADY_ACTIVE(HttpStatus.FORBIDDEN, "Cannot join a room that is already active.");
+    ALREADY_ACTIVE(HttpStatus.FORBIDDEN, "This room has already started; please wait for the host to play again.");
```

Comment:
Not sure why, but my frontend still displays the old message (see image below). This isn't a huge issue so I say merge it anyway if this issue is unique to me, but if something else has to be added to make this change, it might be a good idea to do so in this PR.

<img width="1266" alt="Room Already Active Issue" src="https://user-images.githubusercontent.com/7987279/112555827-2de44200-8d86-11eb-896f-db3078cc6762.png">

Reply:
Good catch - there's a custom string on the fronted on the join page that sets this custom error message - I updated that to be the same message

Reply:
Awesome thanks!